lang/en/docs/tutorials/dft/electronic/hubbard.md

```diff
@@ -0,0 +1,44 @@
+# DFT+U and Hubbard parameter calculation in Quantum Espresso
+
+In this tutorial we demonstrate how to perform DFT+U calculation, followed by
+calculation of Hubbard parameter using Quantum Espresso simulation engine on our
+web platform.
+
+!!!warning
+    Here we will follow the latest DFT+U syntax and method introduced in Quantum
+    Espresso version `7.1`. The new template (syntax) **pw_scf_dft_u** is only
+    available to Quantum Espresso version `7.1` or above. If the user would like
+    to use old syntax, please select Quantum Espresso version `7.0` or below,
+    and use **pw_scf_dft_u_legacy**.
+
+1. Navigate to workflow page from the sidebar, and create new workflow. Expand
```

Comment:
Let's add a bit more explanation, headers as breakpoint, and visuals similar https://deploy-preview-262--mat3ra.netlify.app/tutorials/dft/electronic/density-of-states/